build.gradle

```diff
@@ -85,6 +92,6 @@ checkstyle {
 tomcat {
     httpProtocol = 'org.apache.coyote.http11.Http11Nio2Protocol'
     ajpProtocol  = 'org.apache.coyote.ajp.AjpNio2Protocol'
-    httpPort = 8080
+    httpPort = 8099
```

Comment:
Why do you need to change port?

Reply:
I changed it because my server application uses this port.